### PR TITLE
MGMT-9313: Sorting by nodes and adding a sortable creation date in cluster list

### PR DESCRIPTION
**Pull request by ammont82**
Related to https://issues.redhat.com/browse/MGMT-9313

### CODE REVIEW

**Comment by ammont82** [comment]
/hold
this is a 2.6 feature, please dont merge yet

File: frontend/src/routes/Infrastructure/Clusters/ManagedClusters/ManagedClusters.tsx [hunk 1]
```diff
@@ -548,3 +554,23 @@ export function useClusterNodesColumn(): IAcmTableColumn<Cluster> {
         },
     }
 }
+
+export function useClusterCreatedDateColumn(clusterDeployments: ClusterDeployment[]): IAcmTableColumn<Cluster> {
```
Comment (jtomasek) [inline frontend/src/routes/Infrastructure/Clusters/ManagedClusters/ManagedClusters.tsx:558]: Would it be simpler to add the desired `creationTimeStamp` property to the `Cluster` type which is generated from various resources in `get-cluster.ts` module? We could avoid having to reach for `ClusterDeployments` in this component and most likely the sorting would be as simple as `sort: 'creationTimeStamp'`.

What do you think?

**Review by jtomasek — CHANGES_REQUESTED** [review]

File: frontend/src/routes/Infrastructure/Clusters/ManagedClusters/ManagedClusters.tsx (hunk 1 above)
Comment (ammont82) [reply frontend/src/routes/Infrastructure/Clusters/ManagedClusters/ManagedClusters.tsx]: Yes, it works better in this way. I've changed it

**Review by jtomasek — APPROVED** [review]
/lgtm

**Pull request merged**